cookbooks/fb_apache/recipes/default.rb

```diff
@@ -81,6 +97,16 @@
   notifies :restart, 'service[apache]'
 end
 
+# Remove default sites
+node['fb_apache']['default_sites'].each do |site|
```

Comment:
This isn't runtime safe and doesn't follow our model. If people can't modify it after this cookbook, it won't fly.